dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.security-2.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.security-2.0
+IBM-API-Package: com.ibm.wsspi.security.tai; type="ibm-api", \
+ com.ibm.wsspi.security.token; type="ibm-api", \
+ com.ibm.wsspi.security.auth.callback; type="ibm-api", \
+ com.ibm.wsspi.security.common.auth.module; type="ibm-api", \
+ com.ibm.websphere.security.auth.callback; type="ibm-api"
+-features=com.ibm.websphere.appserver.ssl-1.0, \
+ com.ibm.websphere.appserver.securityInfrastructure-1.0, \
+ com.ibm.websphere.appserver.builtinAuthorization-1.0, \
+ com.ibm.websphere.appserver.ltpa-1.0, \
+ io.openliberty.builtinAuthentication-2.0, \
```

Comment:
The builtin authentication feature was named io.openliberty.builtinAuthentication-2.0. If we would rather remove the private security-2.0 feature, I can rename that builtinAuthentication-2.0 feature to be com.ibm..... Your call.

Reply:
I can live with it the way it is for now.  You can do it in the follow on feature.  Since it is a private feature I think you can rename builtinAuthentication-1.0 to io.openliberty as well.

Reply:
I am going to need to change builtinAuthorization in the next feature (jaac / Jakarta Authorization) so I can handle this there if you are alright with it.

If I have to do another build, then I will do it under this one. So i will leave this open until the build results come back.

Reply:
I am fine with waiting for jaac feature.

Reply:
Took this as an action item and will address it in the authorization PR.